.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
This should be in your `.gitignore`. It is used in development so no need to push it up. 
Anytime you have development set-up such as traspilation you should specify what another developer needs to do in order to set up the environment a readme.md.